internal/database/online/dynamodb/stream.go

```diff
@@ -11,6 +11,12 @@ import (
 )
 
 func (db *DB) PrepareStreamTable(ctx context.Context, opt online.PrepareStreamTableOpt) error {
+	// dynamodb is no such thing as a "column", so if you need to add columns, do not do anything.
```

Comment:
dynamodb has no "column", so we do nothing to "add column". see https://stackoverflow.com/a/25610645/16428442

Reply:
fixed